docs/en/Contributing-Getting-Started.md

```diff
@@ -0,0 +1,261 @@
+and find one that is unassigned. You can press the link **Assign to me** in order to assign the
+ticket to yourself. You should assign a ticket to yourself before you start working on it, so others
+in the community know you are working on the ticket.
+
+Notice that all Alluxio JIRA tickets are named by **ALLUXIO-####** where **####** is a number. This
+name will be important later, when you create a pull request. The title of the pull request for this
+ticket will be titled like, **[ALLUXIO-XXXX] Awesome Feature**.
+
+## Creating a Branch in your Clone
+
+After you have taken ticket, go back to the terminal, and go to the directory of your local clone.
+Now, you can start working on the fix!
+
+In order to submit a change to Alluxio, it is best practice to do all of your changes for a single
+JIRA ticket, in its own branch. Therefore, the following will show you how to create a branch.
+
+First, make sure you are on the `master` branch in your clone. You switch to your `master` branch
+with:
+
+```bash
+$ git checkout master
+```
+
+Then, you should make sure your `master` branch is in sync with the latest changes from the evolving
+Alluxio code base. You pull in all the new changes in the project with the following command:
+
+```bash
+$ git pull upstream master
+```
+
+This will pull in all the changes from the Alluxio open source project, into your local `master`
+branch.
+
+Now, you can create a new branch in order to work on the **New Contributor** task you took earlier.
+To create a branch name **awesome_feature**, run:
+
+```bash
+$ git checkout -b awesome_feature
+```
+
+This will create the branch, and switch to it. Now, you can modify the necessary code to address the
+JIRA ticket.
+
+## Creating Local Commits
+
+As you are addressing the ticket, you can create local commits of your code. This can be useful for
+when you have finished a well-defined portion of the change. You can usually stage a file for commit
+with:
+
+```bash
+$ git add <file to stage>
+```
+
+Once all the appropriate files are staged, you can create a local commit of those modifications
+with:
+
+```bash
+$ git commit -m “<concise but descriptive commit message>”
+```
+
+If you want more details, please visit [instructions on how to create commits](https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository).
+
+## Sending a Pull Request
+
+After you have finished all the changes to address the JIRA ticket, you are ready to submit a pull
+request to the Alluxio project! Here are [detailed instructions on sending a pull request](https://help.github.com/articles/using-pull-requests/), 
+but the following is a common way to do it.
+
+First, you have to push all of your local change to your repository in GitHub. After you have
+created all necessary local commits, you can push all your commits to your repository in GitHub.
```

Comment:
nit: remove repetition of "push all your local commits to your repository in GitHub"